### ISSUE TITLE
[BUG] -Created events are not displayed to other users

### ISSUE THREAD

**Issue by Priyaravi8** [rokwire/illinois-app#3632]
**Describe the bug**
The event created by the admin is not displayed to the other users 

**To Reproduce**
1. Install the Illinois app
2. Navigate to the Event feed and tap on it
3. Create an event named "Fun" by "User A"
4. The event is successfully created
5. "User B" search for the Event "Fun"

**Actual Result**
The Event "Fun" is not displayed to the user

**Expected behavior**
The event "Fun" should display to the user B 

**Smartphone (please complete the following information):**
 - Device: [e.g. Android and iphone]
 - Version [e.g. 5.0.52]

**Comment by mihail-varbanov** [comment]
I was also not able to find the event named "Fun" but this is not an indicator for a problem. The event may not be available for the auditory if it is already over, if it is private, if it is not published. Is it sure that your event "Fun" matches these requirements?

Today I needed to test event registration so I asked tnb@illinois.edu to create an event with in-app registration. I ([varbanov@illinois.edu](mailto:varbanov@illinois.edu)) was able to locate and open it (please see the screenshot below). So unless you provide a recreation scenario I am unable to locate the problem, if it persists at all. If you manage to recreate this again please provide a screenshot of the event detail panel, that is launched right after its creation.

![Simulator Screen Shot - iPhone 14 Pro Max - 2023-08-24 at 15 31 23](https://github.com/rokwire/illinois-app/assets/5631487/dc47b942-6548-4cad-a1aa-528f6fe4848f)

**Comment by mihail-varbanov** [comment]
I just noticed that the version used to create the "Fun" event is 4.0.52. In 4.0.53 we introduced a new event property called "published". If it is not initialized it is assumed to be false and the event is not available to the public. Version 4.0.52 does not initialize this property and this explains why it is not visible for other users than the event admin. The resolution is using version 4.0.53 or newer version either to edit event properties and turn on the published flag (in "Publish this event" toddle at the bottom of the panel), or to create a new test event that have it turned on.
